src/main.py

```diff
@@ -0,0 +1,56 @@
+# OMIT !/usr/bin/env python3
+from pathlib import Path
+import click
+from io import StringIO
+
+from src.option_select import select
+from src.run_cocci import run_cocci
+
+@click.command
+@click.option(
+    "-i",
+    "--input",
+    type=Path,
+    required=True,
+    help="Directory containing your C source files.",
+)
+@click.option(
+    '--scope',
+    type=click.Choice(['all', 'select']), 
+    prompt='Choose an option to compare all or just a single file'
+)
+@click.option(
+    "-v",
```

Comment:
This is not a high priority now, but the standard way of doing this would be to turn `-v` into a flag. So, if the command is called without `-v`, that means that more verbose logging is not necessary. having to specify 0 or 1 is not very user-friendly